### ISSUE TITLE
Admin set participants and workflow depositing roles duplicate each other

### ISSUE THREAD

**Issue by geekscruff** [samvera-labs/hyrax#1093]
There is no need to add a ‘depositing’ workflow role if the user is a depositor participant in the admin set. A user MUST be a participant in an admin set (either individually or by group) order to deposit to it. 

Therefore the depositing workflow role seems redundant.

@mjgiarlo thinks this may be in another ticket but we haven't been able to find it

**Comment by atz** [comment]
Just 321 open tickets to read through, right?

**Comment by mjgiarlo** [comment]
Yeah, AFAICT, the ticket I had in mind has disappeared into the ether.

**Comment by jcoyne** [comment]
If they aren't granted the depositing workflow role, then the user won't have the correct permissions to advance the work though the workflow transitions.

**Comment by mjgiarlo** [comment]
@jcoyne I may have misunderstood the intent of this ticket. What I thought @geekscruff was suggesting was that adding a user or group to the Admin Set Depositors should have the effect of adding the `depositing` workflow role to each user or group automatically. I'm not sure that's happening atm. That'd happen around [here](https://github.com/samvera/hyrax/blob/master/app/forms/hyrax/forms/permission_template_form.rb#L145)?
